frontend/src/lib/format/zaaktype.ts

```diff
@@ -1,6 +1,8 @@
+import { getUUIDFromString } from "~/lib/format/string.ts";
+
 /**
  * Splits `obj.url` into parts, using the last part as UUID.
  * @param obj - An object which implements a `url` key (e.g. a Zaaktype).
  */
 export const getZaaktypeUUID = (obj: { url: string }) =>
-  obj.url.split("/").reverse()[0];
+  getUUIDFromString(obj.url);
```

Comment:
Hebben ze niet allemaal een `uuid` attribuut gekregen nu?

Hmmm. Niet als we de antwoorden van OZ 1-op-1 doorgeven. Wat misschien nog ergens gebeurt.